stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.31.1
+version: 1.33.0
```

Comment:
If this gets merged next, this should be 1.32.0.

Reply:
Fixed.